docs/source/developer/how_to_code.rst

```diff
@@ -0,0 +1,69 @@
+.. Describes how to do code
+
+.. _HowToCode:
+
+How to Code
+===========
+
+Before you start contributing in this project please read and accept our :ref:`C4`.
+
+Before you start coding always remember the *Zen of Python* by Tim Peters. If you don't know it just
+type :code:`import this` in an interactive Python interpreter.
+
+We strongly encourage you to use any Linux environment to code. The project was never tested on 
```

Comment:
".. Linux environment or WSL on Windows .."?

Reply:
I treat WSL also as "any Linux"

Reply:
I thought perhaps we should point this out, but if this is clear to Windows users, its ok for me :-)